src/doc/rustc/src/SUMMARY.md

```diff
@@ -23,6 +23,7 @@
     - [armv6k-nintendo-3ds](platform-support/armv6k-nintendo-3ds.md)
     - [armv7-unknown-linux-uclibceabi](platform-support/armv7-unknown-linux-uclibceabi.md)
     - [armv7-unknown-linux-uclibceabihf](platform-support/armv7-unknown-linux-uclibceabihf.md)
+    - [\*-android(eabi)?](platform-support/android.md)
```

Comment:
This is a really minor nit, but while this document uses `*` (which is a fairly widely understood wildcard), it doesn't use any form of regular expression syntax. Can I suggest just using literally `*-android and *-androideabi`?

```suggestion
    - [\*-android and \*android-eabi](platform-support/android.md)
```

Reply:
Done